src/program.js

```diff
@@ -167,6 +167,11 @@ Example: $0 --help run.
         demand: true,
         type: 'string',
       },
+      'timeout' : {
+        describe: 'A length in millesconds to give up after if the add-on ' +
+                  'hasn\'t been signed',
```

Comment:
maybe you need to add `default: undefined` to the yargs config but I'm not sure.